packages/venia-ui/lib/components/MiniCart/ProductList/item.js

```diff
@@ -60,7 +63,12 @@ const Item = props => {
                         root: classes.thumbnail
                     }}
                     width={100}
-                    resource={product.thumbnail.url}
+                    resource={
+                        (configurableThumbnailSource === 'itself' &&
```

Comment:
Similarly, the ternary operator here will help.

Reply:
Done